### PR TITLE
WIP: Make Ceph Health recovery faster

### PR DESCRIPTION
**Pull request by prsurve**

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -1487,11 +1493,17 @@ def finalizer():
                     else:
                         ceph_health_check_base()
                         log.info("Ceph health check passed at teardown")
+                        log.info("Removing Faster recovery profile")
+                        ceph_cmd = "ceph config set osd osd_mclock_profile balanced"
```
Comment (petr-balogh) [inline tests/conftest.py:1497]: You can rename it to `ceph_remove_cmd`, move above line 1470 and remove the duplicated line from 1505.

Comment (prsurve) [reply tests/conftest.py:1497]: sure will do

**Review by petr-balogh — APPROVED** [review]

**Comment by prsurve** [comment]
Running Verification job here https://url.corp.redhat.com/58819bf

**Comment by prsurve** [comment]
tier 4b:- https://url.corp.redhat.com/fdcf9c7
tier 4a:- https://url.corp.redhat.com/449cff4

**Comment by sseshasa** [comment]
@prsurve From the changes, my understanding is that "high_recovery_ops" mclock profile is enabled for the duration of the recovery and once the health is reported 'OK', the profile is reverted back to the default. Is my understanding correct?

I don't see any issue with how the profile is being utilized.

**Pull request closed without merging**